src/applications/check-in/day-of/pages/TravelReview.jsx

```diff
@@ -38,31 +34,44 @@ const TravelQuestion = props => {
     dispatch(recordAnswer({ 'travel-question': 'no' }));
     goToNextPage();
   };
+  const agreementLink = e => {
+    e.preventDefault();
+    jumpToPage('/travel-agreement');
+  };
+  const startOverAction = e => {
+    e.preventDefault();
+    jumpToPage('/travel-question');
```

Comment:
sorry, I wasn't super clear here. This should go back to the TravelQuestion page, but the URL is `travel-pay`, this results in an error because there isn't a `travel-question` URL

Reply:
I thought I was taking crazy sauce yesterday when I was trying that out